Base/Python/tests/test_sitkUtils.py

```diff
@@ -0,0 +1,49 @@
+import slicer
+#import SimpleITK as sitk
+import sitkUtils as su
+
+import unittest
+
+class SitkUtilsTests(unittest.TestCase):
+
+    def runTest(self):
```

Comment:
So, I can erase runTest ? What about setUp and tearDown ? Must I delete them also ?